code/gateway/deregister.q

```diff
@@ -0,0 +1,52 @@
+
+\d .gw
+
+deregcheckfreq:@[value;`.gw.deregcheckfreq;0D00:00:10];
+
+//overwrite this method to not upsert null handles??
```

Comment:
is this a question? I think this is potentially confusing to future devs etc.

Reply:
resolved